master/buildbot/scripts/upgrade_master.py

```diff
@@ -86,10 +86,12 @@ def sighandler(signum, frame):
         """.split())
         print(msg % signum)
 
+    prev_handlers = {}
```

Comment:
this is unrelated to super()?

Reply:
Completely unrelated, this is why I put it in a separate commit